client/src/components/mapping/TabColumnPicking/StaticValueForm/index.tsx

```diff
@@ -34,24 +44,47 @@ interface Props {
   attribute: Attribute;
 }
 
+interface Source {
+  id: string;
+  name: string;
+  template: {
+    name: string;
+  };
+  resources: Resource[];
+}
+
 const StaticValueForm = ({ attribute }: Props) => {
   const dispatch = useDispatch();
 
   const toaster = useSelector((state: IReduxStore) => state.toaster);
-  const { resource } = useSelector((state: IReduxStore) => state.selectedNode);
+  const { source, resource } = useSelector(
+    (state: IReduxStore) => state.selectedNode
+  );
   const attributesForResource = useSelector(
     (state: IReduxStore) => state.resourceInputs.attributesMap
   );
 
+  // If the current attribute is the identifier of the resource and not a Reference identifier
+  const isIdentifierSystem = /^identifier\[\d+\]\.system$/.test(attribute.path);
```

Comment:
you should put that part in the Attribute class from fhir.ts and not use a regex here